backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/question/domain/Question.java

```diff
@@ -314,4 +314,8 @@ public void remove() {
         getTopics().forEach(topic -> topic.getQuestions().remove(this));
         getTopics().clear();
     }
+
+    public boolean isSubmittedQuestion() {
+        return !Arrays.asList(Question.Status.AVAILABLE, Question.Status.DISABLED, Question.Status.REMOVED).contains(status);
```

Comment:
not sure about the performance of this compared with return status == IN_REVISION || status == ...